### DISCUSSION TITLE
Why is it shudown?

### DISCUSSION THREAD

**Discussion by Cuteistfox** [zlatinb/muwire#179] [Q&A]
WHy?

**Comment by zlatinb** [comment]
A variety of non-technical factors influenced my decision.  The time and effort required to maintain the necessary infrastructure, hosting and such was overwhelming but more importantly I suspect that there had been a security breach in my Apple Developer account (I used a Mac to work on MuWire).

I've been thinking about resurrecting MuWire at least partially.  I don't expect to resurrect the websites muwire.com or muwire.i2p but I see from the GItHub traffic stats that occasionally people clone the source code, so I may resume running the discovery server. That way if someone goes through the effort of building MuWire from source they will at least be able to connect to a small network.  If that works out, I may start publishing binaries on GitHub.
